src/scripts/content/teamwork.js

```diff
@@ -150,7 +150,10 @@ togglbutton.render(
     const link = togglbutton.createTimerLink({
       className: 'teamwork',
       buttonType: 'minimal',
-      description: () => getNameHolder().textContent.trim()
+      description: () => getNameHolder().textContent.trim(),
+      projectName: () => {
+        return $('.w-header-titles__project-name a').textContent.trim();
+      }
```

Comment:
Can we by the way make it more failsafe and fallback to empty strings if elements not found? 🙏